store/ssmstore.go

```diff
@@ -32,12 +32,13 @@ var _ Store = &SSMStore{}
 // SSMStore implements the Store interface for storing secrets in SSM Parameter
 // Store
 type SSMStore struct {
-	svc      ssmiface.SSMAPI
-	usePaths bool
+	svc         ssmiface.SSMAPI
+	usePaths    bool
+	kmsKeyAlias string
 }
 
 // NewSSMStore creates a new SSMStore
-func NewSSMStore(numRetries int) (*SSMStore, error) {
+func NewSSMStore(numRetries int, kmsKeyAlias string) (*SSMStore, error) {
```

Comment:
I wanted to make `--kms-key-alias=alias/keyname` work for the SSM backend too, because I thought it would be really confusing if as a user of that backend, I set that flag and it wasn’t used at all.

In this PR I’ll return an error if `--kms-key-alias` is used with the SSM backend. I’ll then make a follow up PR.

Reply:
Awesome, thanks, that sounds good. Yeah, I always felt that the env var was an anti-pattern. https://github.com/segmentio/chamber/wiki/Style-Guide#avoid-environment-variables-in-library-code